src/script_interface/reaction_methods/ReactionEnsemble.hpp

```diff
@@ -39,9 +39,20 @@ class ReactionEnsemble : public ReactionAlgorithm {
   }
 
   void do_construct(VariantMap const &params) override {
+    std::unordered_map<int, double> exclusion_radius_per_type;
+
+    for (auto const &item : get_value_or<std::unordered_map<int, Variant>>(
+             params, "exclusion_radius_per_type", {})) {
+      exclusion_radius_per_type[item.first] =
+          get_value<double>(get_value<std::unordered_map<int, Variant>>(
+                                params, "exclusion_radius_per_type")
+                                .at(item.first));
+    }
```

Comment:
Here and in all the other `do_construct()` methods, how about replacing this logic by the following one-liner?
```c++
   get_map<int, double>(get_value_or<std::unordered_map<int, Variant>>(
      params, "exclusion_radius_per_type", {}))
```